R/html_document.R

```diff
@@ -204,6 +206,7 @@ html_document <- function(toc = FALSE,
 
   # build pandoc args
   args <- c("--standalone")
+  lua_filters <- character(0L)
```

Comment:
Is this in advance for other filter to be added ? If so, then this should be modified line 333
https://github.com/rstudio/rmarkdown/pull/1884/files#diff-d244a3f1d96cb8bba005d659d4cdc69dR333

Reply:
fixed via efcd03ea